### DISCUSSION TITLE
Powerpoint

### DISCUSSION THREAD

**Discussion by orbisvicis** [brunoherbelin/vimix#101] [General]
Can I overlay PowerPoint, or LibreOffice Impress, or in general record video from another application (PowerPoint in presentation mode) with a chroma key for transparency overlays?

**Comment by brunoherbelin** [comment]
Under linux, the screen capture now allows selecting a window:

![Screenshot from 2023-09-19 19-44-51](https://github.com/brunoherbelin/vimix/assets/60393189/b1fc1623-b19b-4717-b23c-685b1e8d3320)

Then, applying a filter like [chormakey](https://github.com/brunoherbelin/vimix/wiki/Filters-and-ShaderToy#alpha) is done by cloning the source and selecting the filter.
